Team news

nathanshammah opened the pull request.

nathanshammah commented:
Trying around how a basic news page and blog post may look like. My html is limited to basic copying from this and qutip webpage.

willzeng commented on CNAME:
```diff
@@ -1 +0,0 @@
-unitary.fund
```
You'll need to leave this in place in order for the site's domain to continue to work with github pages.

nathanshammah replied:
yes sorry, I was trying to render the github pages on my fork to preview it online (as I do [here]( https://nathanshammah.github.io/qutip.github.io/) with this [fork](https://github.com/nathanshammah/qutip.github.io)) but it's fine to look at the html locally only.

willzeng commented on blog.html:
```diff
@@ -0,0 +1,101 @@
+<br>
+<br>
+12, March, 2020<br>
+Author: Nathan<br>
```
"Nathan Shammah" and would be great to link it to your homepage

willzeng commented on blog.html:
```diff
@@ -0,0 +1,101 @@
+12, March, 2020<br>
+Author: Nathan<br>
+<br>
+
```
The below are missing hyperlinks

willzeng commented on blog.html:
```diff
@@ -0,0 +1,101 @@
+        <a href="#grants-made">grants made</a> |
+        <a href="#project-ideas">project ideas</a> |
+        <a href="#faq">faq</a> |
+        <a href="#blog">blog</a>
```
```suggestion
        <a href="blog.html">blog</a>
```

This should link to a page that just has a list of links (like a blog index) and then your post should be on its own separate page linked from there. I can help set this up if you'd like.

nathanshammah replied:
thanks. I am also changing the header menu that was suffering from the same issue.

willzeng commented on jobs/mts.html:
```diff
@@ -37,7 +37,7 @@ <h1>Unitary Fund - Member of Technical Staff (Part-time) </h1>
         open source toolchain for error-mitigated quantum programming. You will develop software and perform research
         as part of the US Department of Energy funded TEAM collaboration between Unitary Fund, Stanford, Lawrence
-        Livermore National Lab, John’s Hopkins Applied Physics Lab, the University of Chicago, and the University
+        Livermore National Lab, Johns Hopkins Applied Physics Lab, the University of Chicago, and the University
```
nice typo catch

willzeng commented on index.html:
```diff
@@ -375,6 +376,12 @@ <h1>Unitary Fund</h1>
+    <p class="subtitle"><a name="news">News</a></p>
+
+    <p>
+        <a href="news.html">Team intro</a>
```
url?

nathanshammah replied:
I'm wondering if this is not pedantic and shall I just remove it.

nathanshammah commented:
Main changes:
- I changed the title to: "Introducing the Technical Team: Unitary Labs"
(could be even "Introducing the Technical Team at Unitary Labs"). 
- Added bold
- cut a part defining UF in the blog post, but added the tagline on the top of the page; seemed odd to introduce UF on its webpage.
- added myself to team 
Wondering if "blog" menu could be "labs" (for future `mitiq` stuff).

willzeng commented:
@nathanshammah I added a blog index (but couldn't push to your fork) and so opened a new PR at #6 Check that one out for review but then should be good to release soon.
